### PR TITLE
Create privacy switches

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- adds a `Privacy` section to the switchboard
- adds 3 CMP-related switches to the new section

this paves the way to removing all the other cmp/tcf/ccpa-related switches already on there in commercial and feature sections.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### CODE REVIEW

File: common/app/conf/switches/PrivacySwitches.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import conf.switches.SwitchGroup.{Privacy}
+import org.joda.time.LocalDate
+
+trait PrivacySwitches {
+
+  val Cmp = Switch(
+    SwitchGroup.Privacy,
+    "consent-management",
+    "Enable consent management. Individual frameworks will also need to be switched on.",
+    owners = group(Commercial),
```
Comment (mxdvl) [inline common/app/conf/switches/PrivacySwitches.scala:14]: Is it really?

File: common/app/conf/switches/PrivacySwitches.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package conf.switches
+
+import conf.switches.Expiry.never
+import conf.switches.Owner.group
+import conf.switches.SwitchGroup.{Privacy}
+import org.joda.time.LocalDate
```
Comment (mxdvl) [inline common/app/conf/switches/PrivacySwitches.scala:6]: Not needed it they don't expire.

**Review by mxdvl — APPROVED** [review]

File: common/app/conf/switches/PrivacySwitches.scala [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import conf.switches.Expiry.never
+import conf.switches.Owner.group
+import conf.switches.SwitchGroup.{Privacy}
+
+trait PrivacySwitches {
+
+  val Cmp = Switch(
+    SwitchGroup.Privacy,
+    "consent-management",
+    "Enable consent management. Individual frameworks will also need to be switched on.",
```
Comment (jfsoul) [inline common/app/conf/switches/PrivacySwitches.scala:12]: What happens to features that need consent when this is turned off? They only work for users who have already consented?

**Review by jfsoul — APPROVED** [review]

File: common/app/conf/switches/PrivacySwitches.scala (hunk 3 above)
Comment (sndrs) [reply common/app/conf/switches/PrivacySwitches.scala:12]: the CMP wouldn't initialise, so there would be no privacy management at all. anything that needs consent would not run. 

not sure why we'd need this normally, but there was no way of disabling it when sourcepoint was down on friday, so it's designed to cover that type of scenario

**Pull request merged**